### DISCUSSION TITLE
After deleting an alert/detection in the UI, how do I clear the corresponding entry and thumbnail from Home Assistant media browser?

### DISCUSSION THREAD

**Discussion by monsterlabbit** [blakeblackshear/frigate#14619] [General]
When I delete a particular alert/detection in the UI, I noticed that the corresponding thumbnail is still showing up in the media browser in Home Assistant under clips as the entry is still in the database and the corresponding thumbnail in /media/frigate/clips/review is not deleted. How do I delete it so it does not show up? I don't want to clear all alerts only selected ones so changing the retention policy or deleting the db file is not ideal.

**Comment by NickM-27** [comment]
These are not connected so they can't be. This will be available in the next version but in general that is a lot of deletions.

**Reply by monsterlabbit** [reply]
What is the rne UI?

**Reply by NickM-27** [reply]
Sorry I meant the next version, 0.15
